.github/workflows/ci.yml

```diff
@@ -85,11 +85,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: config
-      run: ./config enable-asan enable-ubsan no-cached-fetch && perl configdata.pm --dump
+      run: ./config --debug enable-asan enable-ubsan no-cached-fetch no-dtls no-tls1 no-tls1-method no-tls1_1 no-tls1_1-method no-async && perl configdata.pm --dump
```

Comment:
Why all the no-xxx options?  Not testing bits of the code under no-cached-fetch doesn't seem ideal.  Not that there were problems with any of these (that I remember).

Likewise for the additional removed tests below.

Reply:
I added those to speed up the tests a little as the no-caching build is taking the longest time when --debug was added. It also helps testing some other combinations of build options.